src/use_case/recommend_word/RecommendInteractor.java

```diff
@@ -0,0 +1,16 @@
+package use_case.recommend_word;
+
+public class RecommendInteractor implements RecommendInputBoundary {
+    final RecommendOutputBoundary recommendPresenter;
+
+    public RecommendInteractor(RecommendOutputBoundary recommendOutputBoundary, RecommendOutputBoundary recommendPresenter) {
```

Comment:
Why are there two RecommendOutputBoundary parameters initialized with different names here?

Reply:
Nice catch Ali. One of the parameters should be a DataAccessInterface. 

@nothing128 did you do this as a placeholder until the call to the synonyms API was figured out?